app/assets/javascripts/custom.js

```diff
@@ -52,9 +58,19 @@ function createGame(gameParams) {
     type: 'POST',
     url: "/api/v1/games",
     data: gameParams,
+    dataType: "json",
     success: function(newGame) {
       var listItem = "<li><a href='/" + newGame.slug + "'>" + newGame.name + "</a></li>";
       $("#one-game").append(listItem);
     },
+    error: function(xhr) {
+      var errorObject = xhr.responseJSON.errors;
+      var errorKeys = Object.keys(errorObject);
+      for(var i = 0; i < errorKeys.length; i++){
+        errorObject[errorKeys[i]].forEach(function(error) {
+          $("#welcome").append(`<span>${errorKeys[i]}:${error},  </span>`).css('background-color', '#ffcccc');
```

Comment:
Nice use of ES6 string interpolation here. If you're mixing ES5/ES6 though, you might need to add some special stuff to spockets to make it cross-browser compatible...